pkg/reconciler/v1alpha1/taskrun/taskrun_test.go

```diff
@@ -88,9 +89,23 @@ func TestReconcile(t *testing.T) {
 	}{
 		{"success", "foo/test-taskrun-run-success", false, true, ""},
 		// all test modes
-		// create a build
-		// update a status
-		// say finished
+		// taskrun w/o build
+		//  creates a build w/ correct build.Spec
+
+		// task w/ a build that isn't done
+		//   updates status to mirror build
+		//
+		// TODO(aaron-prindle)
+
+		// task w/ a build that is done
+		//   says finished
+		// TODO(aaron-prindle)
+
+		// =================
+		// TODO(aaron-prindle)
+		// taskrun w/ no referenced task
+		//   errors? validation hook?
+
```

Comment:
my general preference would be to avoid committing comments like this unless they are really specific - what about creating an Issue in the backlog instead with this list of cases to add coverage for?

Reply:
removed and added #140